### ISSUE TITLE
JavaScript/TypeScript Support in LangFuse and LangChain for Dataset Experimentation

### ISSUE THREAD

**Issue by guidev** [langfuse/langfuse-js#237]
## Is your feature request related to a problem?

Currently, there is no support for running experiments on datasets using LangChain in JavaScript/TypeScript environments.

## Describe the solution you'd like

I would like to see a JavaScript/TypeScript implementation similar to the following Python functionality:

```
dataset = langfuse.get_dataset("<dataset_name>")

for item in dataset.items:
    # Langchain callback handler that automatically links the execution trace to the dataset item
    handler = item.get_langchain_handler(run_name="<run_name>")
    
    # Execute application and pass custom handler
    my_langchain_chain.run(item.input, callbacks=[handler])

```
## Related sub-libraries

- [ ] All of them
- [ ] langfuse
- [ ] langfuse-node

**Comment by marcklingen** [comment]
Agree, this'd be great. I'll migrate this to our centralized github discussions as we track feature requests there

**Comment by marcklingen** [comment]
Agree, this'd be great. Generally tracking feature requests in our centralized GitHub discussions board but I think this here is very clear and makes sense as an add on to the current implementation of datasets in JS/TS. thanks for raising this
